### DISCUSSION TITLE
Ip Camera for proxmox

### DISCUSSION THREAD

**Discussion by 4358haltern** [aaronwmorris/indi-allsky#1598] [Q&A] [answered]
Hello, I want to use an IP camera in connection with Proxmox. Can you recommend one for me?

**Comment by aaronwmorris** [comment, marked as answer]
This is a tough recommendation, because the camera needs to support accessing the IP Cameras data as a JPEG file.  This is not usually a listed feature, you just have to figure out if the camera supports the option.
 
I had decent luck with Reolink cameras.  I have seen others use Amcrest cameras for this.

**Reply by 4358haltern** [reply]
can you recommend a model❓
